packages/web/src/pages/edit-collection-page/desktop/EditCollectionPage.tsx

```diff
@@ -111,6 +114,7 @@ export const EditCollectionPage = () => {
           initialValues={initialValues}
           onSubmit={handleSubmit}
           isUpload={false}
+          autoFocus={focus}
```

Comment:
nit: feels like this should change names one layer further down

'focus' for the form and 'autoFocus' when it's for this specific field

Reply:
yeah down wit dat